gwcs/tests/test_wcs.py

```diff
@@ -187,6 +189,11 @@ def test_from_fiducial_composite():
     coord_result = w(1, 1, 1, output='numericals_plus')
     assert_allclose(coord_result[0], u.Quantity(11.5 * u.micron))
 
+    # https://github.com/spacetelescope/gwcs/issues/135
+    # 2.1 - trans not quantities, inputs are quantities
+    with pytest.raises(u.UnitsError):
```

Comment:
@nden -- How about something like this? When you fix `gwcs` to handle it in the future, this test would *fail* because the expected error is no longer raised. This is more explicit than `xfail`-ing something that is actually passing. Unfortunately, I don't have a good enough understanding of GWCS transformation to add more than this.